src/kOS/Function/Suffixed.cs

```diff
@@ -405,4 +407,52 @@ public override void Execute(SharedObjects shared)
             shared.Cpu.PushStack(new WaypointValue(point, shared));
         }
     }    
+
+    [Function("transferall")]
+    public class FunctionTransferAll : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            var transferTo = shared.Cpu.PopValue();
+            var transferFrom = shared.Cpu.PopValue();
+            var resourceName = shared.Cpu.PopValue().ToString();
+
+            var resourceInfo = TransferManager.ParseResource(resourceName);
+            if (resourceInfo == null)
+            {
+                throw new KOSInvalidArgumentException("TransferAll", "Resource",
+                    resourceName + " was not found in the resource list");
+            }
+
+            object toPush = shared.TransferManager.CreateTransfer(resourceInfo, transferTo, transferFrom);
+            shared.Cpu.PushStack(toPush);
+        }
+
+    }
```

Comment:
This function will need to be updated to use the new user_functions branch's paradigm.  I didn't do it in user_functions because this branch and that branch were awaiting merge simultaneously, and this function didn't exist yet in develop when I went through and changed them all.